packages/core/src/integrations/extraerrordata.ts

```diff
@@ -92,7 +92,8 @@ export class ExtraErrorData implements Integration {
         for (const key of errorKeys) {
           let value = error[key];
           if (isError(value)) {
-            value = (value as Error).name || (value as Error).constructor.name;
+            const name = (value as Error).name || (value as Error).constructor.name;
+            value = `${name}: ${(value as Error).message}`;
```

Comment:
```suggestion
            const message = (value as Error).message ? `: ${(value as Error).message}` : '';
            value = `${name}${message}`;
```
To make sure it's not `Error: undefined`